lesson_07/homeworks_07.py

```diff
@@ -68,4 +94,51 @@ def find_substring(str1, str2):
 """  Оберіть будь-які 4 таски з попередніх домашніх робіт та
 перетворіть їх у 4 функції, що отримують значення та повертають результат.
 Обоязково документуйте функції та дайте зрозумілі імена змінним.
-"""
\ No newline at end of file
+"""
+# task 7
+"""  Написати функцію, яка обчислює суму двох чисел.
+"""
+def continuation_numbers(number_1,number_2):
+    return number_1 + number_2
```

Comment:
```suggestion
def continuation_numbers(number_1,number_2):
    return number_1 + number_2
```
це ж було уже (с) Л.Д.Кучма